### ISSUE TITLE
Can you let me translate your module into Chinese?

### ISSUE THREAD

**Issue by ponyzerteal** [haoming37/TheOtherRoles-GM-Haoming#7]
このモジュールを中国語に翻訳させていただけますか？
We appreciate your contribution to this game, but in China, some players have encountered some troubles when playing your module, one of which is that no one has translated it into Chinese. If the author agrees with me to translate your module, please list some English translations of Japanese occupations, so that I can translate it more in line with the functions of occupations in the game.

私たちはあなたがこのゲームに貢献してくれてありがとうございます。しかし、中国では、一部のプレイヤーがあなたのモジュールを遊んでいる間にいくつかのトラブルに遭遇しました。その中で、大きな点は誰も中国語に翻訳していないことです。もし作者があなたのモジュールを翻訳することに同意してくれるなら、日本語の職業の設定の英語翻訳を並べてください。そうすれば、私が翻訳する時にゲーム内の職業の機能にもっとぴったり合うようになります

**Comment by haoming37** [comment]
@ponyzerteal 
I started writing Descriptions in English as below.
Is this readable?

```
Fox

Fox is neutral role has unique win condition.
His win condtion is to complete tasks and survive.
He has strong wincondition but he can't fix sabotage.
He can create Immoralist from crewmates, Immoralist can help fox to pretend  to fix sabotage. 
(Immoralist wins if Fox wins)

Rule1.
If he fills his win condition, he overwrites others win except other neutral role's solo win.
 (Exceptions: jester,arsonist,vulture,madscientis etc...) 
In other words，Fox cant win until others fills win condition  

Rule2.
Fox has two ability to survive
one is stealth, no one can target fox if he is invisible. 
second one is showing arrows to player who can kill Fox.

Rule3. 
About Immoralist
1. Immoralist dies if fox dies.
2. Immoralist can see flash when other player dead.
3. Immoralist can suicide
4. Immoralist can see arrows to Fox
```

**Comment by ponyzerteal** [comment]
I'm sorry to reply to your message now, because I've been on frequent business trips recently, which has led to unstable online time. First of all, thank you very much for your help. Secondly, I used dnspy to sinicize the other roles before. Because of my poor Japanese level, I can't accurately sinicize your interesting career. Now that I have English introduction, I can translate my career into Chinese very well. Thank you for your attention to our Chinese players.

**Comment by haoming37** [comment]
Updated English Translations to Strings.xlsx.
https://github.com/haoming37/TheOtherRoles-GM-Haoming/blob/translation/Strings.xlsx

If you have any questions feel free to ask here(sorry my english is not easy to read)

**Comment by haoming37** [comment]
English README and In game English translations are added on v 1.11.0
https://github.com/haoming37/TheOtherRoles-GM-Haoming/blob/haoming-main/README_ENG.md

**Comment by ponyzerteal** [comment]
> English README and In game English translations are added on v 1.11.0 https://github.com/haoming37/TheOtherRoles-GM-Haoming/blob/haoming-main/README_ENG.md

Hi haoming, recently, we popularized the module you made in Chinese, and achieved good results and attracted more people's attention. However, I have to say that there are still some problems in translation (our translation work is done by a group, but few people can use json), so I wonder if it is possible to change your subsequent version of the other roles GM into a better Chinese version with dnspy, just like The Other Roles GM-.
 The following is the link of our localization forum.
 https://amongus.freeflarum.com/

**Comment by ponyzerteal** [comment]
Hi haoming, recently, we popularized the module you made in Chinese, and achieved good results and attracted more people's attention. However, I have to say that there are still some problems in translation (our translation work is done by a group, but few people can use json), so I wonder if it is possible to change your subsequent version of the other roles GM into a better Chinese version with dnspy, just like The Other Roles 
 The following is the link of our localization forum.
 https://amongus.freeflarum.com/

**Comment by haoming37** [comment]
Hi ponyzerteal.
The Other Roles GM has easy way to support multi language.
I recommend to use Strings.xlsx instead of dnspy.
https://github.com/haoming37/TheOtherRoles-GM-Haoming/blob/haoming-main/Strings.xlsx
(if you dont have excel, use libre office to open it)
This is list of words in multi language
If you can add Chinese words into Strings.xlsx, you dont need to mod dll with dnspy anymore and every my updates will contain Chinese translations.

this is the example of translation in Chinese. 
https://github.com/yukinogatari/TheOtherRoles-GM/pull/10
sadly, dusk didnt added chinese translation last update.

**Comment by haoming37** [comment]
@ponyzerteal 
If you can fill Chinese translation in Strings.xlsx and give it to me.
I can build TheOtherRolesGM.dll with Chinese translation.

If you have any questions feel free to ask here.

**Comment by ponyzerteal** [comment]
The string.xlsx has been translated, and here is the link.

https://github.com/ponyzerteal/TheOtherRolesGM-H-C/blob/main/Strings.xlsx

**Comment by haoming37** [comment]
New version of the dll with Chinese translation
https://github.com/haoming37/TheOtherRoles-GM-Haoming/releases/download/v1.11.11/TheOtherRolesGM.dll

**Comment by ponyzerteal** [comment]
Look forward to updating

**Comment by haoming37** [comment]
this one works for chinese translation(the last one was corrupted)
https://github.com/haoming37/TheOtherRoles-GM-Haoming/releases/download/v1.11.12/TheOtherRolesGM.dll

**Comment by ponyzerteal** [comment]
Some translation problems (due to previous mistakes in my documents) The following is a screenshot of the problems.
![problems1](https://user-images.githubusercontent.com/97593043/158984718-882d57f3-32e5-45ae-ac73-08388807d119.png)
![problems2](https://user-images.githubusercontent.com/97593043/158984752-e00cad23-11b5-4067-a30a-72505fde5659.png)

1.Trapper should be translated as 设陷者, and the option of the trapper in the setting should be 设陷者陷阱持有数.
2. The last impostor should be translated as 独孤狼, not IL.
If the pictures I uploaded are not loaded, I will upload them again.

**Comment by haoming37** [comment]
I will update new version asap
![image](https://user-images.githubusercontent.com/13807574/158989502-40f98ab8-c0e0-4ea9-975a-75332dd0b313.png)
![image](https://user-images.githubusercontent.com/13807574/158989827-54e0f71a-7c80-4206-bd1f-c5def66bb1c0.png)

**Comment by haoming37** [comment]
i think this works perfectly
https://github.com/haoming37/TheOtherRoles-GM-Haoming/releases/download/v1.11.13/TheOtherRolesGM.dll

**Comment by ponyzerteal** [comment]
Thank you very much, but please also pay attention to rest.

**Comment by ponyzerteal** [comment]
Encountered some bugs and suggestions recently.
1. When Lost Importer is evil guess, there will be problems with the skills of Lost Importer's guesser (the skills of evil guesser and Lost Importer will not be used together)
2.Trapper's trap suggestion is set so that his teammates can see his trap.
3. On the issue of successful settlement, the crew (especially the shifter) often appears in the successful settlement of the spy, which is not a big bug, but I still hope you can fix it.
4. By the way, can lost imposter use his guess skill more than once in a meeting (if he can guess more than once in the option setting)

**Comment by haoming37** [comment]
- When Lost Importer is evil guess, there will be problems with the skills of Lost Importer's guesser (the skills of evil guesser and Lost Importer will not be used together)
I know about this issue.
I plan to fix it

- 2.Trapper's trap suggestion is set so that his teammates can see his trap.
Impostor and Fox can see traps
![image](https://user-images.githubusercontent.com/13807574/159163306-f27babed-8ab9-4877-bc95-38998c14efa3.png)

- On the issue of successful settlement, the crew (especially the shifter) often appears in the successful settlement of the spy, which is not a big bug, but I still hope you can fix it.
sorry i dont understand this situation.
 
- By the way, can lost imposter use his guess skill more than once in a meeting (if he can guess more than once in the option setting)
 yes, if setting is on in guesser settings

**Comment by haoming37** [comment]
fixed last impostor and evil guesser bugs

**Comment by haoming37** [comment]
https://github.com/haoming37/TheOtherRoles-GM-Haoming/releases/tag/v1.12.2

**Comment by ponyzerteal** [comment]
I'm sorry to reply to your new message recently due to work reasons. Besides, I have received some feedback on questions and a notice that you have updated your new role. The following is the most feedback problem in the play: the owner will drop the line inexplicably, which is the problem I received the most. The server is sub-service. I hope you can tell me the solution. 
Finally, I hope you can give me the latest string document and I will improve the localization of new roles. I wish you all the best

**Comment by ZeMingoh233** [comment]
[Strings.xlsx](https://github.com/haoming37/TheOtherRoles-GM-Haoming/files/8509518/Strings.xlsx)
Sorry for my long in coming. Here's our translation of v2.0.5.

**Comment by no1xsyzy** [comment]
> [Strings.xlsx](https://github.com/haoming37/TheOtherRoles-GM-Haoming/files/8509518/Strings.xlsx) Sorry for my long in coming. Here's our translation of v2.0.5.

LGTM

**Comment by ZeMingoh233** [comment]
[Strings.xlsx](https://github.com/haoming37/TheOtherRoles-GM-Haoming/files/8601719/Strings.xlsx)
Hi there. Here's our translation for GMH v2.0.10. Thank you for your work.

**Comment by ZeMingoh233** [comment]
[Strings.xlsx](https://github.com/haoming37/TheOtherRoles-GM-Haoming/files/8746064/Strings.xlsx)
I completed the new strings' translation and optimize the former texts.

**Comment by ZeMingoh233** [comment]
[Strings.xlsx](https://github.com/haoming37/TheOtherRoles-GM-Haoming/files/8748744/Strings.xlsx)
Jekyll and Hyde's part is compeleted.
What's more, I've changed some of my former translation texts, so please use the table directly;)

**Comment by haoming37** [comment]
updated with attached Strings.xlsx
https://github.com/haoming37/TheOtherRoles-GM-Haoming/releases/tag/v2.1.55

**Comment by ZeMingoh233** [comment]
[strings.xlsx](https://github.com/haoming37/TheOtherRoles-GM-Haoming/files/9032974/strings.xlsx)
Here's the translation for the latest version of strings.
